packages/design-system-docs/src/pages/components/ChoiceList/checkbox-radio.example.html

```diff
@@ -86,7 +86,7 @@
   </fieldset>
   <div class="example--wrapper example--inverse">
     <fieldset class="ds-c-fieldset">
-      <legend class="ds-c-label"><span>Inverse example</span></legend>
+      <legend class="ds-c-label ds-c-label--inverse"><span>Inverse example</span></legend>
```

Comment:
Is this because the CSS is off in IE11? None of the other inverse labels have this class, I'd prefer to add the inverse label to all of them, or none of them. For this PR, can we avoid making this change, and then we can handle it in the future?

Reply:
This is only for the html example, without `ds-c-label--inverse`, label text is black instead of white on ie11. I noticed that the components correctly applies these classes `"ds-c-label ds-c-label--inverse"` for inverse label.

Reply:
Right, thats what I assumed. I think for this PR we should remove this change due to the reasons I mentioned above. In this case I think its better to not fix it than to only fix one label. You are welcome to create a followup PR that fixes this for all labels, but dont include it here.

Reply:
Great, will remove this fix and add a followup PR for fixing the html examples with inverse label.